commitlint.config.js

```diff
@@ -0,0 +1,11 @@
+export default {
+  rules: {
+    'header-case': [ 2, 'always', ['sentence-case']],
+    'header-max-length': [ 2, 'always', 50 ],
+    'header-full-stop': [ 2, 'never', '.' ],
+    'body-case': [ 2, 'always', ['sentence-case']],
+    'body-leading-blank': [ 2, 'always' ],
+    'body-max-line-length': [ 2, 'always', 72 ],
```

Comment:
I am not a big fan of limiting line length for the body. For example, it forces to break URLs. What is the reasoning behind this limitation? It's been a long time since terminals were only 80 characters wide 😉

Reply:
I can remove this rule, I implemented it because it is explicitly written [in our contributing file](https://github.com/OpenTermsArchive/engine/blob/main/CONTRIBUTING.md#commit-messages). If we decide to remove this rule, I'll ensure our contributing guidelines are updated accordingly.

Reply:
Very good point! I'm personally in favor of not limiting, but that can wait until my next commit with frustratingly too short body lines 😉 I'll let you decide if you want to implement that here or not.